feature/search/src/main/java/com/google/samples/apps/nowinandroid/feature/search/navigation/SearchNavigation.kt

```diff
@@ -28,10 +28,18 @@ fun NavController.navigateToSearch(navOptions: NavOptions? = null) {
     this.navigate(searchRoute, navOptions)
 }
 
-fun NavGraphBuilder.searchScreen(onBackClick: () -> Unit) {
+fun NavGraphBuilder.searchScreen(
+    onBackClick: () -> Unit,
+    onInterestsClick: () -> Unit,
+    onTopicClick: (String) -> Unit = {},
```

Comment:
NIT: add `/* no-op */` to empty function

Reply:
> NIT: add `/* no-op */` to empty function

Reply:
Done.